visual-dotnet/SauceLabs.Visual/BaselineOverride.cs

```diff
@@ -0,0 +1,32 @@
+using SauceLabs.Visual.GraphQL;
+using SauceLabs.Visual.Models;
+
+namespace SauceLabs.Visual
+{
+    public class BaselineOverride
```

Comment:
Can we add a summary what it is? Since this is a public interface, we need to provide some explanation what it does

Reply:
Done!